apps/davi/public/locales/es/translation.json

```diff
@@ -0,0 +1,435 @@
+    "tokenAmount": "Cantidad de Tokens",
+    "errorLoadingTreasuries": "Error al cargar tesorerías",
+    "noTreasuryAssets": "No hay activos de tesorería disponibles"
+  },
+  "actionBuilder": {
+    "action": {
+      "actions_one": "Acción",
+      "actions_other": "Acciones",
+      "addAction": "Añadir Acciones",
+      "saveAction": "Guardar Acción",
+      "onChainActions_zero": "No hay acción on-chain",
+      "onChainActions_one": "{{count}} acción on-chain",
+      "onChainActions_other": "{{count}} acciones on-chain",
+      "unknownAction": "Acción Desconocida",
+      "onchainAction": "Acción On-chain",
+      "noOnchainAction": "No hay acción on-chain",
+      "confirmRemoval": "Confirmar Eliminación",
+      "areYouSureYouWantToRemoveAction": "¿Está seguro de que desea remover esta acción?",
+      "theRemovalCannotBeReverted": "La eliminación no puede ser revertida",
+      "removeAction": "Remover Acción",
+      "remove": "Remover",
+      "edit": "Editar"
+    },
+    "options": {
+      "against": "En contra de",
+      "for": "A favor de",
+      "addOption": "Añadir Opción",
+      "editOption": "Editar Opción",
+      "deleteOption": "Borrar Opción",
+      "optionLabel": "Etiqueta de la Opción",
+      "option": "Opción {{optionKey}}",
+      "options": "Opciones"
+    },
+    "inputs": {
+      "amount": "Cantidad",
+      "inWei": "Valor en wei",
+      "asset": "Activo",
+      "token": "Token",
+      "tokenAmount": "Cantidad de Tokens",
+      "recipient": "Destinatario",
+      "ethereumAddress": "Dirección Ethereum",
+      "toAddress": "A la Dirección",
+      "address": "Dirección",
+      "contract": "Contrato"
+    },
+    "permissions": {
+      "permissions": "Permisos",
+      "anyAddress": "Cualquier Dirección",
+      "maxValue": "Valor Máximo",
+      "setPermissions": "Establecer Permisos",
+      "setPermissionsFor": "Establecer permisos para",
+      "setApprovalPermissionsFor": "Establecer permisos de aprobación para",
+      "setTransferPermissionsFor": "Establecer permisos de transferencia para",
+      "permissionDenied": "Permiso denegado",
+      "permissionDeniedMessage": "El DAO no tiene permiso para realizar esta acción",
+      "warningMessage": "Advertencia",
+      "proposalNotExecuted": "Esta propuesta no puede ser ejecutada. ¿Está seguro de que desea continuar?",
+      "function": "Función",
+      "functionName": "Nombre de la Función",
+      "functionSignature": "Firma de la Función",
+      "assetTransfer": "Transferencia de Activo",
+      "functionCall": "Llamada a la Función",
+      "functionCalls": "Llamadas a la Función",
+      "noPermissionsSet": "No hay permisos establecidos",
+      "dataNotAvailable": "Datos de permisos no disponibles",
+      "allowedAmount": "Cantidad permitida"
+    },
+    "transfer": {
+      "transfer": "Tranferir",
+      "unknownToken": "Token desconocido"
+    },
+    "repMint": {
+      "mint": "Acuñar",
+      "mintRep": "Acuñar REP",
```

Comment:
Just an opinion. I was wondering if we should use just "Crear" or simply leave the word "mint" since is a very standard word in this space and specific to the mint function in the rep token contract.